trade-finance-manager-ui/server/controllers/utilisation-reports/helpers/reconciliation-summary-helper.ts

```diff
@@ -42,19 +55,39 @@ export const getDueDateText = (reportDueDate: Date) => {
   return `Reports${reportIsPastDue ? ' were ' : ' '}due to be received by ${formattedReportDueDate}.`;
 };
 
-export const getReportPeriodHeading = (submissionMonth: IsoMonthStamp, reportPeriods: ReportPeriod[]) => {
+const getReportPeriodHeadingWithAllPeriods = (submissionMonth: IsoMonthStamp, reportPeriods: ReportPeriod[]) => {
   const isCurrentSubmissionMonth = isSameMonth(new Date(submissionMonth), new Date());
 
   const formattedReportPeriods = reportPeriods.map((reportPeriod) => getFormattedReportPeriodWithShortMonth(reportPeriod, true)).join(' and ');
 
   return `${isCurrentSubmissionMonth ? 'Current reporting period' : 'Open reports'}: ${formattedReportPeriods}`;
 };
 
+const getReportPeriodHeadingWithPeriodEnd = (submissionMonth: IsoMonthStamp, reportPeriods: ReportPeriod[]) => {
+  const isCurrentSubmissionMonth = isSameMonth(new Date(submissionMonth), new Date());
+
+  const formattedReportPeriodEnd = format(getDateFromMonthAndYear(reportPeriods[0].end), 'MMMM yyyy');
+
+  return `${isCurrentSubmissionMonth ? 'Current' : 'Open reports'} reporting period end: ${formattedReportPeriodEnd}`;
```

Comment:
Not sure if the constants already exists for these?

Reply:
No, I don't believe so

Reply:
Can they please be created and used to keep it DRY. Thanks

Reply:
I'm a little uncertain what a good name for this would be. It's only used and expected to be used in this one place, what do you think of this?
```
export const BANK_REPORTS_FOR_PERIOD_TABLE_HEADER_PREFIX = {
  CURRENT_REPORTING_PERIOD: 'Current',
  NOT_CURRENT_REPORTING_PERIOD: 'Open reports',
} as const;
```
So then it would become:
`${isCurrentSubmissionMonth ? BANK_REPORTS_FOR_PERIOD_TABLE_HEADER_PREFIX.CURRENT_REPORTING_PERIOD : BANK_REPORTS_FOR_PERIOD_TABLE_HEADER_PREFIX.NOT_CURRENT_REPORTING_PERIOD}`

Reply:
Hey @RLavender98 above snippet looks good thank you.